From 975e5ce8ac04632606917e321cddad4bcce31553 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Thu, 28 Sep 2017 12:02:03 +0200
Subject: [PATCH] gtkwindow: Don't force enable CSD under Windows

CSD look a bit out of place under Windows, especially
when the application doesn't use the headerbar and the CSD
doesn't add any value.

Instead let the application decide, like under X11.
It can still be force enabled through GTK_CSD=1
---
 gtk/gtkwindow.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/gtk/gtkwindow.c b/gtk/gtkwindow.c
index 6c5f136043..d178390536 100644
--- a/gtk/gtkwindow.c
+++ b/gtk/gtkwindow.c
@@ -6119,12 +6119,6 @@ gtk_window_should_use_csd (GtkWindow *window)
     }
 #endif
 
-#ifdef GDK_WINDOWING_WIN32
-  if (g_strcmp0 (csd_env, "0") != 0 &&
-      GDK_IS_WIN32_DISPLAY (gtk_widget_get_display (GTK_WIDGET (window))))
-    return TRUE;
-#endif
-
   return (g_strcmp0 (csd_env, "1") == 0);
 }
 
-- 
2.24.0

